docs/publisher/index.rst

```diff
@@ -128,6 +128,35 @@ You can also attach a callback to the future:
     future.add_done_callback(callback)
 
 
+Publish Flow Control
+--------------------
+
+If publishing large amounts of messages or very large messages in quick
+succession, some of the publish requests might time out, especially if the
+bandwidth available is limited. To mitigate this the client can be
+configured with custom :class:`~.pubsub_v1.types.PublishFlowControl` settings.
+
+You can configure the maximum desired number of messages and their maximum total
```

Comment:
Would be good to describe the 3 LimitExceededBehavior options. Otherwise, looks good.